backend/src/email/middlewares/email.middleware.ts

```diff
@@ -241,18 +270,20 @@ const getCustomFromAddress = async (
   res: Response,
   next: NextFunction
 ): Promise<Response | void> => {
-  const email =
-    req.session?.user?.email ||
-    (await AuthService.findUser(req.session?.user?.id))?.email
   const defaultEmail = config.get('mailFrom')
-  const result = []
+  const result = [defaultEmail]
 
-  try {
-    const fromAddress = await CustomDomainService.getCustomFromAddress(email)
-    if (fromAddress) result.push(fromAddress)
-    result.push(defaultEmail)
-  } catch (err) {
-    return next(err)
+  if (!config.get('emailFallback.activate')) {
```

Comment:
One side effect of this is that when email fallback is activated, the email section in the settings tab will be reverted back to the request form (like when they don't have a custom from registered). 

Rather than not returning the custom from address as an option, would it be better to just rely on the error returned when the `PUT` call is made.

Reply:
Sounds good